reporting_app/templates/untabbed_datatables.html

```diff
@@ -20,4 +20,270 @@
+          cumulative_yield_chart.draw(cumulative_run_yield_data, cumulative_run_yield_options);
+
+
+          var histogram_options = {
+          title: 'Yield metrics',
+          legend: { position: 'top', maxLines: 2 },
+          colors: ['#1A8763', '#5C3292'],
+          interpolateNulls: false};
+          var yield_histogram = new google.visualization.Histogram(document.getElementById('yield_histogram'));
+          yield_histogram.draw(histogram_data, histogram_options);
+
+      } else if (pipeline_type == 'samples') {
+
+          var samples_sequenced = {{ samples_sequenced | safe }};
+          samples_sequenced_aggregate = aggregateData(samples_sequenced)
+
+          weeks = samples_sequenced_aggregate[0]
+          months = samples_sequenced_aggregate[1]
+          aggregate_weeks = samples_sequenced_aggregate[2]
+          aggregate_months = samples_sequenced_aggregate[3]
+
+          by_month_samples_data = []
+          by_week_samples_data = []
+
+
+          for (var key in weeks) {
+                if(typeof aggregate_weeks[key] != 'undefined') {
+                      by_week_samples_data.push([key, aggregate_weeks[key]])
+                } else if(typeof aggregate_weeks[key] == 'undefined') {
+                      by_week_samples_data.push([key, weeks[key]])
+                }
+          }
+
+          for (var key in months) {
+                if(typeof aggregate_months[key] != 'undefined') {
+                      by_month_samples_data.push([key, aggregate_months[key][0], aggregate_months[key][1], aggregate_months[key][2]])
+                } else if(typeof aggregate_months[key] == 'undefined') {
+                      by_month_samples_data.push([key, months[key], months[key], months[key]])
+                }
+          }
+
+          var sample_data = new google.visualization.DataTable();
+          sample_data.addColumn('string', 'X');
+          sample_data.addColumn('number', 'First');
+          sample_data.addColumn('number', 'Repeat');
+          sample_data.addColumn('number', 'Total');
+          sample_data.addRows(by_month_samples_data);
+          var sample_options = {
+            hAxis: {title: 'Date'},
+            vAxis: {title: 'Number of Samples'}};
+          var chart3 = new google.visualization.LineChart(document.getElementById('samples_sequenced'));
+          chart3.draw(sample_data, sample_options);
+
+          var histogram_data = google.visualization.arrayToDataTable( {{ hist | safe }});
+          var histogram_options = {
+          title: 'Yield metrics',
+          legend: { position: 'top', maxLines: 2 },
+          colors: ['#1A8763', '#5C3292'],
+          interpolateNulls: false};
+          var chart2 = new google.visualization.Histogram(document.getElementById('yield_histogram'));
+          chart2.draw(histogram_data, histogram_options);
+      }
+}
+
+</script>
+</head>
+
+<body>
```

Comment:
These `div`s should be in the template's `content` block, rather than hard-coded to the `<body>`.